validation-test/stdlib/UnicodeUTFEncoders.swift

```diff
@@ -14,13 +14,12 @@ import StdlibUnittest
 
 import Foundation
 
-protocol TestableUnicodeCodec : UnicodeCodec {
-  associatedtype CodeUnit : FixedWidthInteger
+protocol TestableUnicodeCodec: Unicode.Encoding {
```

Comment:
This file has been updated to use only non-deprecated APIs.

I should probably keep a copy of the old file, to run in compatibility mode.